channels/dz.m3u

```diff
@@ -1,5 +1,5 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="ALANISTV.dz" tvg-name="AL ANIS TV" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/G60iJen.png" group-title="",AL ANIS TV
+#EXTINF:-1 tvg-id="AlAnisTV.dz" tvg-name="Al Anis TV" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/G60iJen.png" group-title="",Al Anis TV [Offline]
 https://cdn02.hta.dz/abr_htatv/El_Fhama_TV/playlist.m3u8
```

Comment:
They've reimplemented the token usage for those streams. Simply do a POST on https://token.kube.easybroadcast.fr/authtoken?url=http://apphta.easybroadcast.fr/apphta, take the wmsAuthSign and paste it next to the link.

Reply:
Doesn't work
![image](https://user-images.githubusercontent.com/7253922/129436014-5acf4bfb-3e4e-4a08-a51d-bc3cea1584aa.png)

Reply:
Interesting... it seems that the majority of their channels are down.